src/main/java/dan200/computercraft/shared/peripheral/speaker/SoundTask.java

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
It wasn't necessary to create a whole new class for this. Use an anonymous class.